ElectronClient/gui/NoteText.jsx

```diff
@@ -876,8 +876,11 @@ class NoteTextComponent extends React.Component {
 			}
 		} else if (urlUtils.urlProtocol(msg)) {
 			if (msg.indexOf('file://') === 0) {
-				// When using the file:// protocol, openExternal doesn't work (does nothing) with URL-encoded paths
-				require('electron').shell.openExternal(urlDecode(msg));
+				if (shim.platformName()  === 'win32') {
```

Comment:
What happens on a 64bit Windows?

Reply:
Hi, shim.platform fetches the platform information from process.platform. The possible return values are listed [here](https://nodejs.org/api/process.html#process_process_platform). So even for a win64 system "win32" is returned.

For more background information see [LINK](https://stackoverflow.com/questions/8683895/how-do-i-determine-the-current-operating-system-with-node-js#8684009)

Reply:
Thanks a lot for the info. It just looked weird to me.